code.gs

```diff
@@ -93,6 +93,13 @@ var indentSize = 4;
 
 // END MANDATORY CUSTOMIZATION
 
+// START OPTIONAL CUSTOMIZATION
+
+// Types of events to report, in a list. Can include one or more of 'BIRTHDAY', 'ANNIVERSARY', 'CUSTOM'
+var eventTypes = ['BIRTHDAY'];
```

Comment:
Yes, I think making the editable part a hash/dict of explicit booleans as you suggest would be more friendly to non-techies (and self-documenting, so the comment-line can be removed). I think the conversion one-liner can be slightly simplified too due to the filter-method coercing the test-retval to boolean anyway, but maybe I'm just showing more familiarity with other languages than JS (please tell me if I'm assuming wrong for JS here). Let me know if the below looks good and I'll update the commit accordingly.

```javascript
// START OPTIONAL CUSTOMIZATION

var eventTypes = {
    BIRTHDAY: true,
    ANNIVERSARY: false,
    CUSTOM: false,
}

// END OPTIONAL CUSTOMIZATION

//// other stuff...

eventTypes = Object.keys(eventTypes).filter(function(x) { return eventTypes[x]; });
```